test/repository_timestamps_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+describe Lotus::Repository do
+  before do
+    adapter, uri, mapper = Lotus::Model::Adapters::SqlAdapter, SQLITE_CONNECTION_STRING, MAPPER
+    ArticleRepository.adapter = adapter.new(mapper, uri)
+  end
+
+  describe 'create timestamp on persisting' do
+    let(:article) { Article.new(title: 'New article with timestamp') }
+    let(:persisted_article) { ArticleRepository.create(article) }
+
+    it '#created_at' do
```

Comment:
What do you think about using `describe` instead of `it` here? And describe what it does with `it`, for example: `it 'touches created_at attribute'`